at_radial_gauges/lib/radial_gauges/_text_annotation_gauge.dart

```diff
@@ -0,0 +1,220 @@
+// import 'dart:math';
+// import 'dart:ui';
+
+// import 'package:flutter/material.dart';
+// import 'package:radial_gauges/utils/utils.dart';
+
+// class TextAnnotationGauge extends StatefulWidget {
+//   /// Creates a Range Pointer Gauge.
+//   ///
+//   /// The [minValue] and [maxValue] must not be null.
+//   TextAnnotationGauge({
+//     this.minValue = 0,
+//     required this.maxValue,
+//     required this.actualValue,
+//     this.label = '',
+//     this.pointerColor = Colors.blue,
+//     this.decimalPlaces = 0,
+//     this.animate = true,
+//     Key? key,
+//   }) : super(key: key);
+
+//   /// Sets the minimum value of the gauge.
+//   double minValue;
+
+//   /// Sets the maximum value of the gauge.
+//   double maxValue;
+
+//   /// Sets the pointer value of the gauge.
+//   double actualValue;
+
```

Comment:
Do we want to keep this commented file ?

Reply:
no, I added it to gitignore for now.